src/Lightning.ts

```diff
@@ -338,14 +353,23 @@ export const LightningMixin = (superclass) => class extends superclass {
           })
         })
         
-        await transactionNotification({ amount: sats, user: payeeUser, hash: id, logger: this.logger, type: "paid-invoice" })
+        transactionNotification({ amount: sats, user: payeeUser, hash: id, logger: this.logger, type: "paid-invoice" })
```

Comment:
Has the effect of removing `await` been tested here?

Reply:
not really but is it necessary?

`transactionNotification` return `Promise<void>` and basically we don't care about the result of this function for what the below this function. 

so awaiting for this is unnecessary. it will increase the response time to the request to the user. the notification can be sent in parallel.

Reply:
Just wondering if the transactionNotification method will still be executed completely if the parent function returns while it's in the middle of completing

Reply:
[Stackoverflow](https://stackoverflow.com/questions/45594596/async-function-without-await-in-javascript) thinks it will complete.
> As you assumed, if no await is present the execution is not paused and your code will then be executed in a non-blocking manner.